packages/playground/cli/src/blueprints-v1/worker-thread-v1.ts

```diff
@@ -178,21 +196,23 @@ export class PlaygroundCliBlueprintV1Worker extends PHPWorker {
 					allow_url_fopen: '1',
 					disable_functions: '',
 				},
-				hooks: {
-					async beforeWordPressFiles(php) {
-						mountResources(php, mountsBeforeWpInstall);
-					},
-				},
 				cookieStore: internalCookieStore ? undefined : false,
 				dataSqlPath,
 				spawnHandler: sandboxedSpawnHandlerFactory,
+				async onPHPInstanceCreated(php) {
+					await mountResources(php, mountsBeforeWpInstall);
+					if (wordpressBooted) {
+						await mountResources(php, mountsAfterWpInstall);
+					}
+				},
 			});
 			this.__internal_setRequestHandler(requestHandler);
+			wordpressBooted = true;
 
 			const primaryPhp = await requestHandler.getPrimaryPhp();
 			await this.setPrimaryPHP(primaryPhp);
 
-			mountResources(primaryPhp, mountsAfterWpInstall);
+			await mountResources(primaryPhp, mountsAfterWpInstall);
```

Comment:
I know this won't run twice for the same PHP instance, but it's not obvious on the first glance. Let's add some commentary either here or in line 205 above:

```ts
					if (wordpressBooted) {
						await mountResources(php, mountsAfterWpInstall);
					}
```